### PR TITLE
Fix typos in README.md

### PR DESCRIPTION
**Pull request by sam-sla**
Fix typos in README using the english dictionary from aspell.

### CODE REVIEW

**Comment by frennkie** [comment]
Would you mind squashing these into one commit..?!

File: README.md [hunk 1]
```diff
@@ -40,19 +40,19 @@ The RaspiBlitz is build from the following minimal parts:
...
 * 16GB Micro SD-Card
-* Powersupply >=3A (stable output is important)
+* Power supply >=3A (stable output is important)
 
 **All parts together are at around 150-200 USD - based on shops and location.**
 
 ### Buy a ready-2-go RaspiBlitz (Germany, EU and International)
 
-If you like to support the RaspiBlitz project you can order a ready-2-go RaspiBlitz or a all-you-need-hardwareset for yourself or for your RaspiBlitz workshop from [raspiblitz.com](https://raspiblitz.com) *(pay with Bitcoin/Lightning or with deprecated PayPal/CreditCard)*
+If you like to support the RaspiBlitz project you can order a ready-2-go RaspiBlitz or a all-you-need-hardware set for yourself or for your RaspiBlitz workshop from [raspiblitz.com](https://raspiblitz.com) *(pay with Bitcoin/Lightning or with deprecated PayPal/CreditCard)*
```
Comment (frennkie) [inline README.md:49]: I guess '...hardware-set' would better match the tone here..

**Comment by sam-sla** [comment]
> Would you mind squashing these into one commit..?!

Sure can do that, just thought it would be easier to review. I don't know how this repo is configured, but github has a squash merge commit option, when you merge the PR it squashes it into one commit in master :)

**Comment by sam-sla** [comment]
Done!

**Comment by openoms** [comment]
Thanks you caught a lot with this one! LGTM

**Comment by frennkie** [comment]
>  I don't know how this repo is configured, but github has a squash merge commit option, when you merge the PR it squashes it into one commit in master

You are right.. I could have simply done that. Thanks anyway! :-D

**Review by frennkie — APPROVED** [review]

**Pull request merged**